packages/edit-site/src/components/editor/index.js

```diff
@@ -61,6 +61,14 @@ function Editor( { settings: _settings } ) {
 		};
 	}, [] );
 
+	const skeletonLabels = {
```

Comment:
I wonder if we really need these now, as the default ones apply generically :)

Reply:
Pretty easy to change them later on if needed. I'm going to merge this to avoid more of the rebase hell. 😄

Reply:
Actually scratch that, I updated this in 68c5e2613d6722c435eb3eac112461335b772ecc after reading your comment [here](https://github.com/WordPress/gutenberg/pull/21335#discussion_r402963069).